frontend/src/component/photo/cards.vue

```diff
@@ -159,7 +165,7 @@
             </v-btn>
           </v-img>
 
-          <v-card-actions v-if="photo.Quality < 3 && context === 'review'" class="card-details pa-0">
+          <v-card-actions v-if="!isShredView && photo.Quality < 3 && context === 'review'" class="card-details pa-0">
```

Comment:
I believe this should this be `!isSharedView` instead of `Shred`?

Reply:
fixed it